src/azfilenfs-watchdog

```diff
@@ -0,0 +1,222 @@
+#!/bin/bash
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+#
+# How often does the watchdog look for unmounts and/or to track if stunnel processes are running for
+# nfs file endpoints.
+#
+MONITOR_INTERVAL_SECS=5
+
+#
+# Don't perform mountmap and iptables rule cleanup for unmounted filesystems.
```

Comment:
we dont have ip tables

Reply:
I wanted to add the code related to account migration later, so deleted the code for ip tables from the script. But now, as we will be having common watchdog, the code for ip tables will be resumed.